os/fs/vfs/fs_fstatfs.c

```diff
@@ -0,0 +1,135 @@
+/****************************************************************************
```

Comment:
Samsung License should be added above of nuttx's

Reply:
Done.